requirements/dev-requirements.txt

```diff
@@ -145,7 +155,7 @@ django-formtools==2.3
     #   django-two-factor-auth
 django-logentry-admin==1.0.6
     # via -r base-requirements.in
-django-nose @ https://github.com/dimagi/django-nose/raw/fast-first-1.4.6.1/releases/django_nose-1.4.6.1-py2.py3-none-any.whl
+https://github.com/dimagi/django-nose/raw/fast-first-1.4.6.1/releases/django_nose-1.4.6.1-py2.py3-none-any.whl
```

Comment:
I don't think this should have changed. What version of pip-tools did you use to make requirements? I have version 6.6.0

Reply:
Yep. I had an older version of pip-tools. Fixed it https://github.com/dimagi/commcare-hq/pull/31584/commits/9653a10c4258708538534ccd25de8c06c8d043cf